setup.py

```diff
@@ -4,7 +4,7 @@
 
 install_requires = [
     "gitpython",
-    "typer<0.6",
+    "typer",
```

Comment:
I checked and it works starting from `0.4.1`
```suggestion
    "typer>=0.4.1",
```